src/commands/local/up.ts

```diff
@@ -74,8 +90,51 @@ export default class Up extends BaseCommand<typeof Up> {
     await envConfig.$commit();
     setEnvConfig(envConfig);
 
-    const { flags } = await initCli(this, await this.parse(Up));
+    const { flags, maybeFluenceConfig } = await initCli(
+      this,
+      await this.parse(Up),
+    );
+
     const providerConfig = await initNewReadonlyProviderConfig({});
+
+    if (flags.reset) {
+      const dirPath = dockerComposeDirPath();
+
+      try {
+        await dockerCompose({
+          args: ["down"],
+          flags: {
+            v: true,
+          },
+          printOutput: true,
+          options: {
+            cwd: dirPath,
+          },
+        });
```

Comment:
Seems like useless nesting. Probably we can improve that a bit with a helper function `noop`?
```suggestion
        await dockerCompose({
          args: ["down"],
          flags: {
            v: true,
          },
          printOutput: true,
          options: {
            cwd: dirPath,
          },
        }).catch(noop);
```

Reply:
I think this one is more of a matter of taste. There are a number of places in CLI code where such pattern is used. It's easy to change later if we decide to